services/skill/skill_service.py

```
@@ -0,0 +1,355 @@
+            for key, value in kwargs.items():
+                if hasattr(skill, key):
+                    if key == 'name' and value:
+                        value = sanitize_input(value)
+                    setattr(skill, key, value)
+            
+            db.session.commit()
+            
+            # Refresh lookup cache
+            self.lookup_service.refresh()
+            
+            return True, skill, None
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)